src/test/java/io/strimzi/utils/k8s/HelmClient.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.utils.k8s;
+
+import io.strimzi.utils.executor.Exec;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static java.util.Arrays.asList;
+
+public class HelmClient {
+    private static final Logger LOGGER = LogManager.getLogger(HelmClient.class);
+
+    private static final String HELM_CMD = "helm";
+    private static final String HELM_3_CMD = "helm3";
```

Comment:
AFAIR this was part of Strimzi STs because we supported helm2 and helm3, is this still relevant?

Reply:
Nope, good catch